src/api/status/src/views/builds.hbs

```diff
@@ -29,10 +18,21 @@
           </li>
         </ul>
       </div>
+      <nav aria-label="breadcrumb">
+        <ol class="breadcrumb bg-transparent mb-0 pb-0 pt-1 px-0 me-sm-6 me-5">
+          <li class="breadcrumb-item text-sm">
+            <a class="opacity-5 text-dark" href="javascript:;">Pages</a>
+          </li>
+          <li class="breadcrumb-item text-sm text-dark active" aria-current="page">
+            Build Log
+          </li>
+        </ol>
+        <h6 class="font-weight-bolder mb-0">Build Log</h6>
+      </nav>
     </div>
   </nav>
-  <div class="container-fluid py-2">
-    <div id="build-header" class="card mb-3 bg-white">
+  <div style="width: 100%; max-height: 100%; ">
```

Comment:
Does `container-fluid` do the same thing as `width: 100%`?

Reply:
container-fluid has other css I did not need, so I added width 100% on its own back since I didnt want the other css.